islandora_usage_stats.install

```diff
@@ -623,6 +672,52 @@ function islandora_usage_stats_update_7002() {
     ),
   );
 
+  $schema['islandora_usage_stats_object_ds_download_log'] = array(
+    'description' => 'This table is used to store the usage information for Fedora datastreams.',
+    'fields' => array(
+      'id' => array(
+        'type' => 'serial',
+        'not null' => TRUE,
+      ),
+      'pid_id' => array(
+        'description' => 'PID ID of the object.',
+        'type' => 'int',
+        'not null' => TRUE,
+      ),
+      'dsid' => array(
```

Comment:
We should create a datastream table containing attributes such as we have for objects.